controllers/callhomeconfig_controller.go

```diff
@@ -189,8 +193,12 @@ func (r *CallHomeConfigReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 			ctrl.Result{})
 	}
 	newSecr := corev1.Secret{
+		Type: corev1.SecretType(core.SecretTypeServiceAccountToken),
 		ObjectMeta: metav1.ObjectMeta{
 			Name: chc.Spec.TargetSecretName,
+			Annotations: map[string]string{
```

Comment:
This looks wrong. This secret is for storing a kubeconfig, not a service account token.